docs/recipes/php_composer.md

```diff
@@ -0,0 +1,73 @@
+# 🐘 Using semantic-release to release a PHP/Composer package
+
+You can use semantic-release to release a [Composer](https://getcomposer.org/) package to [packagist.org](https://packagist.org/).
+Composer works with GitHub repositories, so all that needs to happen is tag a new release on GitHub.
+The last release can be figured out by looking at the git tags of the repository.
+
+### Example annotated `package.json`
```

Comment:
`semantic-release` now support various config files. As php project typically do not use `package.json` maybe it would be better to make the `semantic-release` config in `releaserc.yml` or `releaserc.json` ?

Reply:
Not sure what the advantage of that would be. You still need to install semantic-release, so you need to declare the dependency. You could just install it imperatively in CI of course but then you can't use Greenkeeper/dependencies.io/Renovate to update your deps. And given PHP is primarily used for web applications I think PHP people are familiar with npm/package.json for frontend frameworks.

Reply:
That's a good occasion to mention that other type of config files can be used.
`semantic-release` could be used with `npx semantic-release`, which avoid having to go through the `npm install` step.
Greenkeeper can also be used with `npx` I think. I don't know about renovate.

That also avoid to configure all the mandatory fields in `package.json`.

Reply:
Yeah it's a good idea to mention it.
How would Greenkeeper+npx work? Do you have an example?

Reply:
Greenkeeper no longer needs to install any dependency, it works as a GitHub app that needs to be installed on the repo. It does require a `package.json` file though. But GK is only relevant for node apps

Reply:
Yeah, that's what I meant. Just using `npx` in .travis.yml without package.json won't work with updates.

I disagree that it is only relevant for Node apps. Any user who uses semantic-release may want to auto-update semantic-release. There are also tools like dependencies.io or dependabot that work for both Node/npm and PHP/Composer, so there would be no reason not to use it for both in a project.

So imo using a package.json is a better practice to recommend.

Reply:
okay, I’m lacking the experience with non-Node projects, I feel like we are guessing here :) let’s go ahead with it and see how folks adapt it and what trouble they run in 👍